<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>997</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background: transparent;
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../imagem/icons/back.png</normaloff>../imagem/icons/back.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="frame_4">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>90</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>70</width>
         <height>90</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>1000</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../imagem/MOTORMAX-LGO.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>1000</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>90</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>890</width>
         <height>70</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>890</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font-size: 22px;
margin-top: 30px;
font: plain;
}</string>
       </property>
       <property name="text">
        <string>Cadastrar novo funcionario</string>
       </property>
      </widget>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>1200</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1200</width>
        <height>600</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 240);
border-radius: 15px;
border: 1px solid rgba(0, 0, 0,0.1); </string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>520</y>
         <width>150</width>
         <height>45</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #f59e0b;
color: black;
border: none;
border-radius: 8px;
padding: 12px;
font-size: 14px;
font-weight: bold;
min-height: 20px;
}

QPushButton:hover{
background-color: #d97706;
}

QPushButton:pressed{
background-color: #b45309;
}

QPushButton:disabled{
background-color: #582503;
}</string>
       </property>
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>75</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>12</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string>Nome *</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>520</y>
         <width>150</width>
         <height>48</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: transparent;
color: Black;
border: 2px solid;
border-radius: 8px;
padding: 12px;
font-size: 14px;
font-weight: bold;
min-height: 20px;
}

QPushButton:hover{
background-color: rgb(234, 227, 174);
}

QPushButton:Pressed{
background-color: rgb(210, 203, 150);
}</string>
       </property>
       <property name="text">
        <string>Excluir</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>100</y>
         <width>420</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit[echoMode=&quot;2&quot;], QLineEdit[echoMode=&quot;0&quot;] {
                border: 2px solid #e5e7eb;
                border-radius: 8px;
                padding: 12px;
                font-size: 14px;
                color: #374151;
            }
QLineEdit:focus{
border: 2px solid #7f8082;
}

QLineEdit:hover{
background-color: rgb(234, 236, 240);
}</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>Ex: Cleiton@gmail.com</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>175</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>12</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string>CPF *</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>75</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>12</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string>Email *</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>100</y>
         <width>420</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit[echoMode=&quot;2&quot;], QLineEdit[echoMode=&quot;0&quot;] {
                border: 2px solid #e5e7eb;
                border-radius: 8px;
                padding: 12px;
                font-size: 14px;
                color: #374151;
            }
QLineEdit:focus{
border: 2px solid #7f8082;
}

QLineEdit:hover{
background-color: rgb(234, 236, 240);
}</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>Ex: Cleiton</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>200</y>
         <width>420</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit[echoMode=&quot;2&quot;], QLineEdit[echoMode=&quot;0&quot;] {
                border: 2px solid #e5e7eb;
                border-radius: 8px;
                padding: 12px;
                font-size: 14px;
                color: #374151;
            }
QLineEdit:focus{
border: 2px solid #7f8082;
}

QLineEdit:hover{
background-color: rgb(234, 236, 240);
}</string>
       </property>
       <property name="inputMask">
        <string>999.999.999-99</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>000.000.000-00</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
